config/app/csp.yaml

```diff
@@ -0,0 +1,16 @@
+parameters:
+  csp_parameters:
+    default-src: "'none'"
+    script-src: "'self' 'unsafe-inline' 'unsafe-eval' https://cdn.matomo.cloud/histologe.matomo.cloud/matomo.js https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/ https://cdn.jsdelivr.net/npm/leaflet.markercluster@1.5.3/dist/ https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.8/dist/ https://cdn.jsdelivr.net/npm/tippy.js@6/dist/ https://cdn.jsdelivr.net/npm/quill@2.0.2/dist/ https://cdn.jsdelivr.net/npm/tinymce@7.1.2/"
+    style-src: "'self' 'unsafe-inline' https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/ https://cdn.jsdelivr.net/npm/leaflet.markercluster@1.5.3/dist/ https://cdn.jsdelivr.net/npm/leaflet.markercluster@1.5.3/dist/ https://cdn.jsdelivr.net/npm/tippy.js@6/dist/ https://cdn.jsdelivr.net/npm/quill@2.0.2/dist/ https://cdn.jsdelivr.net/npm/tinymce@7.1.2/"
+    style-src-attr: "'self' 'unsafe-inline'"
+    img-src: "'self' data: blob: https://voxusagers.numerique.gouv.fr https://*.tile.openstreetmap.org https://cdn.jsdelivr.net https://jedonnemonavis.numerique.gouv.fr"
+    connect-src: "'self' https://api-adresse.data.gouv.fr https://cdn.matomo.cloud https://histologe.matomo.cloud https://koumoul.com https://sentry.incubateur.net"
+    font-src: "'self'"
+    frame-src: "'none'"
+    object-src: "'none'"
+    base-uri: "'self'"
+    form-action: "'self'"
+    frame-ancestors: "'none'"
+    media-src: "'self'"
+    require-trusted-types-for: "'script'"
```

Comment:
j'ai ajouté celui-ci comme conseillé par CSP evaluator, mais c'est un critère expérimental qui n'est pas pris en comptes par tous les navigateurs, sur firefox il ne sert à rien

Reply:
Virer `require-trusted-types-for` qui n'est supporté que par Chrome et Edge et génère plein d'erreurs (et donc plein de taf futur)

Reply:
supprimé